templates/partial/_careers_navigation.html

```diff
@@ -0,0 +1,334 @@
+<div class="dropdown-window-overlay fade-animation"></div>
+<div class="dropdown-window slide-animation">
+	<div class="u-hide p-section" id="careers-menu" aria-hidden="true">
+		<div class="u-fixed-width p-strip">
```

Comment:
Fixed width should be within the strip. Strips add vertical padding

Reply:
I couldn't find a way to do it without breaking the design spacing.